.github/workflows/validate-pr.yml

```
@@ -0,0 +1,21 @@
+name: "Validate PRs"
+
+on:
+  pull_request_target:
+    types:
+      - opened
+      - reopened
+      - edited
+      - synchronize
+
+permissions:
+  pull-requests: read
+
+jobs:
+  validate-pr-title:
+    name: Validate PR title
+    runs-on: ubuntu-latest
+    steps:
+      - uses: amannn/action-semantic-pull-request@v5
```

```suggestion
      - uses: amannn/action-semantic-pull-request@c3cd5d1ea3580753008872425915e343e351ab54  # v5.2.0
```
I'd prefer we pin this to the latest version's commit sha, since this workflow runs in the context of the base branch